.github/PULL_REQUEST_TEMPLATE/release_pull_request_template.md

```diff
@@ -0,0 +1,70 @@
+* [ ] Merge PR to master 
+* [ ] Tag release (on GitHub, make sure to select the release branch as target) and merge back to develop
+
+## Regression Tests
+
+Run all these tests manually on your system. If you succeed, please write a comment with the revisions of the components that you used below. Example: https://github.com/precice/precice/pull/507#issuecomment-530432289
+
+* [ ] SU2 / CalculiX [flap_perp](https://github.com/precice/tutorials/tree/develop/FSI/flap_perp/SU2-CalculiX)
+* [ ] OpenFOAM / OpenFOAM [flow_over_plate](https://github.com/precice/openfoam-adapter/tree/master/tutorials/CHT/flow-over-plate)
+* [ ] OpenFOAM / OpenFOAM - NP mapping in OpenFOAM [flow_over_plate](https://github.com/precice/openfoam-adapter/tree/master/tutorials/CHT/flow-over-plate)
+* [ ] OpenFOAM / OpenFOAM fluid coupling module **???**
+* [ ] OpenFOAM / CalculiX FSI [flap perp](https://github.com/precice/tutorials/tree/develop/FSI/flap_perp/OpenFOAM-CalculiX)
+* [ ] OpenFOAM / CalculiX FSI - NP mapping in CalculiX [3D_Tube](https://github.com/precice/tutorials/tree/develop/FSI/3D_Tube/OpenFOAM-CalculiX)
+* [ ] OpenFOAM / CalculiX / OpenFOAM CHT [heat_exchanger](https://github.com/precice/tutorials/tree/develop/CHT/heat_exchanger/buoyantSimpleFoam-CalculiX)
+* [ ] OpenFOAM / deal.II [flap_perp_2D](https://github.com/precice/tutorials/tree/develop/FSI/flap_perp_2D/OpenFOAM-deal.II)
+* [ ] OpenFOAM / FEniCS [flap_perp](https://github.com/precice/tutorials/tree/master/FSI/flap_perp/OpenFOAM-FEniCS)
+* [ ] OpenFOAM / FEniCS [flow-over-plate](https://github.com/precice/tutorials/tree/master/CHT/flow-over-plate/buoyantPimpleFoam-fenics)
+* [ ] OpenFOAM / Nutils [flow-over-plate](https://github.com/precice/tutorials/tree/master/CHT/flow-over-plate/buoyantPimpleFoam-nutils)
+* [ ] FEniCS / FEniCS [partitioned-heat](https://github.com/precice/tutorials/tree/master/HT/partitioned-heat/fenics-fenics)
+* [ ] ExaFSA: Ateles / FASTEST
+* [ ] Alya
+* [ ] 1D-ElasticTube [C++](https://github.com/precice/elastictube1d/tree/develop/cxx)
+* [ ] 1D-ElasticTube [Python](https://github.com/precice/elastictube1d/tree/develop/python)
+* [ ] SuperMUC
+* [ ] Solverdummy [C++](https://github.com/precice/precice/tree/develop/tools/solverdummies/cpp)
+* [ ] Solverdummy [C](https://github.com/precice/precice/tree/develop/tools/solverdummies/c)
+* [ ] Solverdummy [Fortran](https://github.com/precice/precice/tree/develop/tools/solverdummies/fortran)
+* [ ] Solverdummy [Fortran 2003](https://github.com/precice/precice/tree/develop/tools/solverdummies/f2003)
+* [ ] Solverdummy [Python](https://github.com/precice/python-bindings/tree/develop/solverdummy)
+* [ ] Solverdummy [MATLAB](https://github.com/precice/matlab-bindings/tree/develop/solverdummy)
+
+
+## Post-release
+* [ ] Generate packages
+   * [ ] Ubuntu 18.04
+   * [ ] Ubuntu 19.10
+   * [ ] Arch Linux AUR Package
+* [ ] Update Spack recipe
+* [ ] Send email and do marketing
+* [ ] Tweet
+* [ ] Format the code base
```

Comment:
If we want to keep this in a regular basis, we should move it in the beginning, before testing.

Reply:
Fixed https://github.com/precice/precice/pull/509/commits/b3cba80002021c939e09faebe306835cae45c4da